cmd/mirror-url.go

```diff
@@ -140,33 +143,28 @@ func deltaSourceTarget(sourceURL string, targetURL string, isForce bool, isFake
 				TargetContent: targetContent,
 			}
 		case differInSecond:
-			if isRemove {
-				// todo(nl5887): I'd all force and fake checks to the the actual mirror / harvest
-				if !isForce && !isFake {
-					// Object removal not allowed if force is not set.
-					URLsCh <- URLs{
-						Error: errDeleteNotAllowed(diffMsg.SecondURL),
-					}
-					continue
-				}
+			if !isRemove && !isFake {
+				// Object removal not allowed if --remove is not set.
 				URLsCh <- URLs{
-					TargetAlias:   targetAlias,
-					TargetContent: diffMsg.secondContent,
+					Error: errDeleteNotAllowed(diffMsg.SecondURL),
```

Comment:
This practically means shows an error when --remove is not passed but an object exists only in target.

Reply:
I mean we should not show an error in this case, we should continue.

Reply:
We should @vadmeste otherwise users will not know that they can pass `--remove` it is complicated to not allow that. 

Without this warning message users have no way to know that delete is disallowed or that we found extraneous files on destination.

Reply:
IMO we should throw this warning message such that we can guide the users.

Reply:
I've just tested it.. it is already a WARNING.. not an error.

By the way, errOverWriteNotAllowed and errDeleteNotAllowed errors in typed-errors.go still have --force flag.. that's also need to be changed.

Reply:
LGTM otherwise.

Reply:
Yes will change that @vadmeste